README.md

```diff
@@ -1,5 +1,29 @@
 **DISCLAIMER: THIS IS A WORK IN PROGRESS**
 
+# BigchainDB Enhancement Proposals (BEPs)
```

Comment:
Do we need that top line disclaimer about this being a work in progress?

Reply:
IMO no :smile: